### PR TITLE
Rebuild meet our team

### LINKED ISSUE #86: Rebuild Meet Our Team section
Let's match Meet Our Team section to the Figma design! 
image

A few reminders:
- Please be sure to include the gradient shown 
image I believe this can be done with css' linear-gradient as seen in figma, the green part might be a little tricky
- Please ensure to make this dynamically displayed through perhaps a `map` function instead of copy and duplicating code
- Additionally, please have a single config file that has all the member info on there, the structure can be something like:
```
const teamMembers = {
  execTeam: [{
    name: ...
    position: ...
    photoUrl: ...
    linkedInUrl: ...
  }, ...],
  devTeam: [{
    name: ...
    position: ...
    photoUrl: ...
    linkedInUrl: ...
  }, ...],
  ...
};
```

- Please chat with our PM Emily Wang with how we want to display all the members within the team. The way figma has it shows all the members in the same page, this is not going to be scalable as we grow (e.g. we'll end up with a giant running scroll of a bunch of people's faces. Some ideas: make it tabular, or div with a specified height that's scrollable. 
- Additionally, I do think the linkedin icon linking is useful for visitors to connect with the people on the team, please also chat with Emily on if we want to keep it or make names links to linkedin profiles

Let me know if any other clarifications are needed!

### PR DESCRIPTION
**Pull request by JamesCahyadi**
Rebuild the meet our teams section to look like figma. Also replaced the names and images of the people to represent the current W21 team. Close #86.

### CODE REVIEW

**Review by kerry918 — COMMENTED** [review]
I think it would be better to leave a bit of space between the 'upcoming events' section and the 'meet our team' section. Maybe Jack and Hyunzee could give some suggestions. 

Otherwise, looks great to me!

**Comment by JamesCahyadi** [comment]
> I think it would be better to leave a bit of space between the 'upcoming events' section and the 'meet our team' section. Maybe Jack and Hyunzee could give some suggestions.
> 
> Otherwise, looks great to me!

Yeah I was wondering the same at first too. I added some space lmk if you think it's too much/little space.

**Review by kerry918 — APPROVED** [review]
LGTM!

**Comment by jack-lin-jql** [comment]
A few stylistic change suggestions:
Perhaps add a box shadow to each box to make it more obvious that they are split by card and each card is scrollable?
For example: 
image
CSS Styles:
image
```
Container div:
    border: 1px solid #d4d4d4;
    box-shadow: 2px 3px #d4d4d4;
    padding: 4px;
    border-radius: 8px;
```
```
Card title:
    border-bottom: 2px solid #16212d57;
    margin-left: 8px;
    margin-right: 8px;
    margin-top: 4px;
```

2. As mentioned during the dev sync, perhaps we can move the linkedin icon to beside the name? I tired this out and it looks a bit odd, so feel free to leave it as is and if it bugs anyone enough in the future, we can revisit

**Comment by jack-lin-jql** [comment]
For the static images,
1. Can we please move them under assets/images?
2. Can we group them into subfolders for each team so we have no confusions in the future? E.g. /assets/images/orgTeam/dev -> contains all dev images

**Comment by jack-lin-jql** [comment]
Also, Diana is still our advisor btw, so don't think we should remove her

File: src/components/landing_page/config.js [hunk 1]
```diff
@@ -0,0 +1,272 @@
+// Pictures
```
Comment (jack-lin-jql) [inline src/components/landing_page/config.js:1]: Nice!

File: src/components/landing_page/config.js [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+import Matthew from "./static/images/Matthew.jpg";
+import Noor from "./static/images/Noor.jpg";
+import Pamela from "./static/images/Pamela.jpg";
+import Rifa from "./static/images/Rifa.jpg";
+import Ryan from "./static/images/Ryan.jpg";
+import Salma from "./static/images/Salma.jpg";
+import Sarah from "./static/images/Sarah.jpg";
+import Twinkle from "./static/images/Twinkle.jpg";
+
+export const techPlusTeams = [
```
Comment (jack-lin-jql) [inline src/components/landing_page/config.js:37]: nit: maybe name to techPlusTeamMembers?

**Comment by jack-lin-jql** [comment]
Very nicely done, please take a look at my comments, otherwise, looks great

**Comment by JamesCahyadi** [comment]
Made changes according to Jack's comments. Also changed it so that there is no bottom margin for the people at the bottom so that when you scroll to the bottom, the person's avatar isn't cut off. It looks like this now: 
image

**Comment by jack-lin-jql** [comment]
Hey, one more thing, can we keep the side by side card layout when screen size is around 900px - 1280px? It currently looks odd with a single card as they are super wide:
image

**Review by jack-lin-jql — APPROVED** [review]
Just that one last thing, LGTM otherwise, feel free to merge after making that mentioned change 👍

**Pull request merged**